plugins/source/azure/resources/services/keyvault/keyvault.go

```diff
@@ -50,7 +50,18 @@ func fetchKeyvault(ctx context.Context, meta schema.ClientMeta, parent *schema.R
 		if err != nil {
 			return err
 		}
-		res <- p.Value
+
+		for _, r := range p.Value {
```

Comment:
Can we use a list/get pattern here to ensure concurrent API calls?
See https://github.com/cloudquery/cloudquery/blob/795e40d05ef7cb8f5dbe7b5fa94cb2c02e0f1c16/plugins/source/gcp/resources/services/bigquery/datasets.go#L14